benchmark/merliner/benchmark.py

```diff
@@ -0,0 +1,48 @@
+import subprocess as sp
+
+from datetime import datetime
+
+
+def merlin_cmd(cmd, samples):
+    start_time = datetime.now()
+    merlin_file = open("bench.yaml", "w")
```

Comment:
`with open("bench.yaml", "w") as merlin_file`
with the context manager, you don't need to manually close the file.